azure/eventhub_log_forwarder/activity_log_diagnostic_settings.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "settingName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the diagnostic setting"
+            }
+        },
+        "resourceGroup": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the relevant Resource Group"
+            }
+        },
+        "eventHubNamespace": {
+            "type": "string",
+            "defaultValue": "datadog-eventhub-namespace",
+            "metadata": {
+                "description": "Name of EventHub namespace"
+            }
+        },
+        "eventHubName": {
+            "type": "string",
+            "metadata": {
+                "description": "Optional. Name of the event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category."
+            }
+        }
+    },
+    "variables": {
+        "subscriptionId": "[subscription().subscriptionId]",
+        "eventHubAuthorizationRuleId": "[concat('/subscriptions/', variables('subscriptionId'), '/resourceGroups/', parameters('resourceGroup'), '/providers/Microsoft.EventHub/namespaces/', parameters('eventhubNamespace'), '/authorizationRules/RootManageSharedAccessKey')]"
```

Comment:
Unsure if this matters, but earlier in the json the parameter is called `eventHubNamespace` but here it is `eventhubNamespace`.

Reply:
Nice catch! Thanks!